### DISCUSSION TITLE
AutoMaterialize not working for Airbyte integration

### DISCUSSION THREAD

**Discussion by jonathanneo** [dagster-io/dagster#18212] [Community]
I am defining my airbyte_assets as follows: 

```python
airbyte_assets = load_assets_from_airbyte_instance(
    airbyte=airbyte_instance, 
    connection_to_asset_key_fn=lambda metadata, table_name: AssetKey([metadata.name, "raw", table_name]), 
    connection_to_auto_materialize_policy_fn=lambda _: AutoMaterializePolicy.lazy(),
)
```

However, the automaterialization policy is not being defined: 
![Screenshot 2023-11-22 at 1 57 40 pm](https://github.com/dagster-io/dagster/assets/16749633/22cac9ba-2a79-4be6-ba05-99483ea9fef4)

**Comment by benpankow** [comment]
Hi @jonathanneo, thanks for the report - closing in favor of #18266 this issue
